### PR TITLE
Dont cache objects 2

### PR DESCRIPTION
**Pull request by mnoack**

### CODE REVIEW

**Comment by msmirnoff** [comment]
:+1:

**Comment by akilmadan** [comment]
:+1:

**Comment by mnoack** [comment]
Fair point, will improve the ol-tests.

File: spec/passenger_type_spec.rb [hunk 1]
```diff
@@ -29,4 +29,24 @@
...
+    let(:api) { double }
+
+    before do
+      stub_const('QuickTravel::Api', api)
+      allow(api).to receive(:call_and_validate) { [{id: 1}, {id: 2}] }
+    end
+
+    it 'should cache and fetch again' do
+      QuickTravel::Cache.cache_store.clear
+      all
```
Comment (alxberardi) [inline spec/passenger_type_spec.rb]: To make the spec really clean, this should be in the before block.

**Comment by akilmadan** [comment]
:+1:

**Pull request merged**